docs/src/main/sphinx/release/release-440.md

```diff
@@ -0,0 +1,28 @@
+# Release 440 (21 Feb 2024)
+
+## Delta Lake connector
+
+* Add support for concurrent writes with `INSERT` statements. ({issue}`18506`)
+
+## Hive connector
+
+* Add support for using an HTTP client with the Thrift metastore. ({issue}`20371`)
```

Comment:
do we have docs for that ? Update in https://trino.io/docs/current/admin/properties-http-client.html ? and elsewhere?

Reply:
It's documented alongside the Thrift metastore properties, yeah.